x/simulation/mock_tendermint.go

```diff
@@ -92,7 +92,7 @@ func updateValidators(
 
 		if update.Power == 0 {
 			if _, ok := current[str]; !ok {
-				tb.Fatalf("tried to delete a nonexistent validator: %s", str)
+				continue
```

Comment:
Can you provide background context about this change?

Reply:
There is a scenario in which an error may occur when validators are removed during Simulation, so it was modified to not be an error at this time.